app/views/command-line.html

```diff
@@ -48,7 +53,7 @@ <h1 id="cli">Command Line Tools</h1>
                   <div ng-show="showSessionToken" class="alert alert-warning">
```

Comment:
Might want to show to show this always since I won't see the warning if I click copy to clipboard.